src/test/kotlin/io/github/mojira/arisa/modules/HelpersTest.kt

```diff
@@ -0,0 +1,268 @@
+        val moduleEither = success.right().toFailedModuleEither()
+
+        moduleEither.shouldBeRight(success)
+    }
+
+    "toFailedModuleEither when given list with Throwables in Either should return FailedModuleResponse with all the Throwables in Either" {
+        val throwable1 = Throwable(message = "generic error 1")
+        val throwable2 = Throwable(message = "generic error 2")
+        val throwable3 = Throwable(message = "generic error 3")
+        val moduleEither = listOf(success.right(), throwable1.left(), success.right(), success.right(),
+            throwable2.left(), throwable3.left()).toFailedModuleEither()
+
+        moduleEither.shouldBeLeft(FailedModuleResponse(listOf(throwable1, throwable2, throwable3)))
+    }
+
+    "toFailedModuleEither when given list without Throwables in Either should return ModuleResponse in Either" {
+        val moduleEither = listOf(success.right(), success.right(), success.right()).toFailedModuleEither()
+
+        moduleEither.shouldBeRight(ModuleResponse)
+    }
+
+    "Either.invert when given ModuleResponse in Either should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = ModuleResponse.right().invert()
+
+        moduleEither.shouldBeLeft(OperationNotNeededModuleResponse)
+    }
+
+    "Either.invert when given OperationNotNeededModuleResponse in Either should return ModuleResponse in Either" {
+        val moduleEither = OperationNotNeededModuleResponse.left().invert()
+
+        moduleEither.shouldBeRight(ModuleResponse)
+    }
+
+    "assertContains when given String that contains the other String should return ModuleResponse in Either" {
+        val moduleEither = assertContains("testabctest", "abc")
+
+        moduleEither.shouldBeRight(ModuleResponse)
+    }
+
+    "assertContains when given null should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertContains(null, "abc")
+
+        moduleEither.shouldBeLeft(OperationNotNeededModuleResponse)
+    }
+
+    "assertContains when given empty String should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertContains("", "abc")
+
+        moduleEither.shouldBeLeft(OperationNotNeededModuleResponse)
+    }
+
+    "assertContains when given String that does not contain other strings should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertContains("testabtestab", "abc")
+
+        moduleEither.shouldBeLeft(OperationNotNeededModuleResponse)
+    }
+
+    "assertEmpty when given empty list should return ModuleResponse in Either" {
+        val moduleEither = assertEmpty(listOf<Int>())
+
+        moduleEither.shouldBeRight(ModuleResponse)
+    }
+
+    "assertEmpty when given nonempty list should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertEmpty(listOf<Int>(1337))
+
+        moduleEither.shouldBeLeft(OperationNotNeededModuleResponse)
```

Comment:
super minor but you can use spaces and its more beautiful xd


moduleEither.shouldBeLeft(OperationNotNeededModuleResponse) is the same as 
        moduleEither shouldBeLeft OperationNotNeededModuleResponse

Reply:
ahhhh ok

Reply:
is it kotest only? or how do you know it if works for others?

Reply:
fixed in https://github.com/mojira/arisa-kt/pull/433/commits/c97ce8396473fd6a1d55d8cd40224dfd4c095dec